achievements.lua

```diff
@@ -77,37 +77,81 @@
 	They are to be generated by serializing `module.achievements` along with `module.localData` and copying any images (when we get to those).
 --]]
 
-local local_achievement_file = "Achievements.json"
+-- TODO?: Maybe make importing these 'extra' graphics optional
+import "CoreLibs/graphics"
 
 -- Right, we're gonna make this easier to change in the future.
 -- Another note: changing the data directory to `/Shared/gameID`
 --   rather than the previously penciled in `/Shared/Achievements/gameID`
+local default_shared_achievement_folder <const> = "/Shared/Data/"
+local default_achievement_file_name <const> = "Achievements.json"
+local default_shared_images_subfolder <const> = "AchievementImages/"
+local default_shared_images_updated_file <const> = "_last_seen_version.txt"
```

Comment:
Labeling these with a `default_` prefix seems like poor practice given that once this library is out of the prototyping phase these paths cannot change. Changing them to proper `<const>` variables is fair play, although the overall immutability is why I had simply hard-coded them into the "return a path" functions.

Reply:
Done!